recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   host:
     - python
     - expandvars
-    - setuptools >=47
+    - setuptools
```

Comment:
Not a blocker, but since it was already there, I think it was okay to leave it. However, I do understand trying to satisfy the linter. Lol